elements/urth-viz-explorer/urth-viz-explorer.html

```diff
@@ -56,6 +56,15 @@
                 notify: true,
                 observer: '_typeChanged'
             },
+
+            /**
+             * Allows user to select multiple chart components (bars in barchart, slices in pie chart, etc).
+             */
+            multiSelect: {
```

Comment:
well, explorer has a chart and table by containment and generally doesn't control any options, so not sure it's really necessary or consistent to do this